types/phoenix_live_view/hooks.d.ts

```diff
@@ -0,0 +1,41 @@
+import LiveSocket from "./live_socket";
+
+export type OnReply = (reply: any, ref: number) => any;
+export type CallbackRef = (customEvent: any, bypass: boolean) => string;
+
+export interface ViewHookInterface {
+  el: HTMLElement;
```

Comment:
it seems you removed `viewName: string` - is it no longer part of the interface ?

Reply:
I just did some spelunking in the phoenix_live_view source repository, and `viewName` was dropped as of v0.16, so I was a little late in picking that up!